pkgs/by-name/wo/world-wall-clock/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "world-wall-clock";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ddelabru";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-OTBYSStCFBrZ8JutrhyyJpi7vRvBAUK0EKTtjvbi13I=";
+  };
+
+  build-system = with python3.pkgs; [ poetry-core ];
+
+  dependencies = with python3.pkgs; [
+    tzdata
+    urwid
+    xdg-base-dirs
+  ];
+
+  nativeCheckInputs = with python3.pkgs; [ pytestCheckHook ];
+
+  pytestFlagsArray = [ "tests/*" ];
+
+  meta = {
+    description = "A TUI application that provides a multi-timezone graphical clock in a terminal environment.";
```

```suggestion
    description = "TUI application that provides a multi-timezone graphical clock in a terminal environment";
```
https://nixos.org/manual/nixpkgs/stable/#var-meta-description